### PR TITLE
German language files

### PR DESCRIPTION
**Pull request by DrDeath**

### CODE REVIEW

**Comment by DrDeath** [comment]
German language files should be done now.

**Comment by DrDeath** [comment]
sorry, forgot to set the correct file permission

File: language/de/activitystats.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		2		=> '%d Gäste',
+	),
+
+	'ACTIVITY_STATS_WORD'				=> ' und',
+	'ACTIVITY_STATS_EXP'				=> 'Die folgenden Daten basieren auf den heutigen Aktivitäten',
+	'ACTIVITY_STATS_EXP_TIME'			=> array(
+		0		=> 'Die folgenden Daten basieren auf den Benutzer Aktivitäten gerade jetzt', // h=m=s=0
+		1		=> 'Die folgenden Daten basieren auf den Benutzer Aktivitäten seit ', // first non zero value is 1
+		2		=> 'Die folgenden Daten basieren auf den Benutzer Aktivitäten seit ', // first non zero value is >1
+	),
```
Comment (RobertHeim) [inline language/de/activitystats.php]: ACTIVITY_STATS_DAYS missing

**Comment by DrDeath** [comment]
should be done now....

**Comment by RobertHeim** [comment]
Thank you!

**Pull request merged**